docs/deploy-models-using-seldon.md

```diff
@@ -0,0 +1,52 @@
+# Deploy models on OpenShift using Seldon Server
+
+Have a model trained and ready to share with the world? In this guide, we go over how to deploy models as a microservice using one of Seldon Core’s pre-packaged [model servers](https://docs.seldon.io/projects/seldon-core/en/v1.1.0/servers/overview.html).
+
+## Pre-requisites
+
+* An ML model trained using one of these supported ML [libraries](https://docs.seldon.io/projects/seldon-core/en/latest/nav/config/servers.html) or a custom model (guide forthcoming) .
+* OpenShift namespace with [seldon operator installed](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#openshift) and administrator privileges available.
```

Comment:
Is there anything we can add here about, using existing OPF resources? like "(see below for instructions on accessing the opf seldon instances)" or something?

Reply:
let me see if we have any existing Seldon related documentation on opf

Reply:
Could not find any existing access docs on seldon. Issue -https://github.com/operate-first/apps/issues/1413

Reply:
```suggestion
* OpenShift namespace with [seldon operator installed](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#openshift) and administrator privileges available. (If you are using the [Operate First cluster](https://console-openshift-console.apps.zero.massopen.cloud/), you can use the namespace `ds-ml-workflows-ws` which already has Seldon deployed) 
```

Reply:
changes made :)